src/Libraries/Microsoft.Extensions.AI/ChatCompletion/DistributedCachingChatClient.cs

```diff
@@ -56,6 +65,13 @@ public JsonSerializerOptions JsonSerializerOptions
         set => _jsonSerializerOptions = Throw.IfNull(value);
     }
 
+    /// <summary>Gets or sets additional cache key values used to inform the key employed for storing state.</summary>
+    public IReadOnlyList<object>? AdditionalCacheKeyValues
```

Comment:
@eiriktsarpalis, are you good with this public API?

Reply:
How about `CacheKeyDefaultAdditionalValues`? Kind of long but it both identifies its purpose and communicates its relationship to the `additionalValues` parameter.

Reply:
If it's clear to users that this is meant to augment the `additionalValues` parameter, I'm fine with leaving `Default` out of the name.